src/components/container/ConnectFitbit/ConnectFitbit.stories.tsx

```diff
@@ -38,4 +38,16 @@ export const NotEnabledHide = Template.bind({});
 NotEnabledHide.args = { previewState: "notEnabled", title: "Fitbit", disabledBehavior: "hide" };
 
 export const NotEnabledDisplayError = Template.bind({});
-NotEnabledDisplayError.args = { previewState: "notEnabled", title: "Fitbit", disabledBehavior: "displayError" };
\ No newline at end of file
+NotEnabledDisplayError.args = { previewState: "notEnabled", title: "Fitbit", disabledBehavior: "displayError" };
+
+export const HideConnectedFetchComplete = Template.bind({});
+HideConnectedFetchComplete.args = { previewState: "fetchComplete", title: "Fitbit", hideWhenConnected: true };
+
+export const HideConnectedNotConnected = Template.bind({});
+HideConnectedNotConnected.args = { previewState: "notConnected", title: "Fitbit", hideWhenConnected: true };
```

Comment:
I don't think this is needed bc it is covered by the Not Connected story

Reply:
Same feeling on HideConnectedUnauthorized

Reply:
These stories have been combined